app/src/main/res/layout-land/onboarding_slide_final.xml

```diff
@@ -45,8 +45,7 @@
         android:fontFamily="sans-serif-medium"
```

Comment:
In this view, you can remove `android:layout_height="wrap_content"` considering it is already defined in styles.xml
Check: `Heading1 -> TextView.Common1`